backend/api/offenderSearchApi.js

```diff
@@ -8,19 +8,31 @@ const offenderSearchApiFactory = client => {
   }
 
   const changeFieldNames = () => data =>
-    data.map(({ uiId, prisonerNumber, bookingId, firstName, lastName, dateOfBirth, prisonId, prisonName, status }) => ({
-      uiId,
-      offenderNo: prisonerNumber,
-      firstName,
-      lastName,
-      dateOfBirth,
-      latestBookingId: Number(bookingId),
-      latestLocationId: prisonId,
-      latestLocation: prisonName,
-      currentlyInPrison: status && status.startsWith('ACTIVE') ? 'Y' : 'N',
-      currentWorkingFirstName: firstName,
-      currentWorkingLastName: lastName,
-    }))
+    data.map(
+      ({
+        uiId,
+        prisonerNumber,
+        bookingId,
+        firstName,
+        lastName,
+        dateOfBirth,
+        prisonId,
+        locationDescription,
```

Comment:
Does this rename prisonName to locationDescription?

Reply:
prisoner offender search has been changed to have a new field - `locationDescription`.  This code reads that new field and maps it to `latestLocation`

Reply:
prisonName seems to have dropped our of the list above then. Could very well be intentional, just raising in case not.

Reply:
yes that's because we were reading from the `prisonName` but are name reading from `locationDescription` instead.  We still map to the `latestLocation` in both instances